views/session/new.pug

```diff
@@ -3,13 +3,13 @@ extends ../layout
 block content
   h1= title
   p #{title}
-  form(action="#")
+  form(action="/session/new", method="POST")
```

Comment:
why is this going to session/new if you changed it to session? Maybe change everything back to session/new so we can rest =)